ui/dashboards/src/components/Panel/Panel.test.tsx

```diff
@@ -36,23 +28,35 @@ const FAKE_PANEL_PLUGIN: PluginRegistrationConfig<JsonObject> = {
 };
 
 describe('Panel', () => {
+  let props: PanelProps;
+  beforeEach(() => {
+    props = {
+      definition: {
+        display: {
+          name: 'Fake Panel',
+          description: 'This is a fake panel',
+        },
+        kind: 'FakePanel',
+        options: {},
+      },
+    };
+  });
+
   // Helper to render the panel with some context set
   const renderPanel = () => {
     const { addMockPlugin, pluginRegistryProps } = mockPluginRegistryProps();
     addMockPlugin(FAKE_PANEL_PLUGIN);
 
     renderWithContext(
       <PluginRegistry {...pluginRegistryProps}>
-        <Panel definition={TEST_DEFINITION} />
+        <Panel {...props} />
       </PluginRegistry>
     );
   };
 
-  it('should render Panel', async () => {
+  it('should render name and info icon', async () => {
     renderPanel();
-
-    const chart = await screen.findByRole('figure');
-    expect(chart).toBeInTheDocument();
-    expect(chart).toHaveTextContent('FakePanel chart');
+    await screen.findByText('Fake Panel');
+    await screen.findByTestId('InformationOutlineIcon');
```

Comment:
Two quick nits here:

1. Same comment as above about whether this can be sync instead of async since you already did an async call right before this.
2. Any way to do this without test IDs (ideally maybe with a Role query)? Test IDs are really a last ditch thing that I'd love if we largely try to avoid like we have at Chronosphere. Totally cool if not (and I know you had no way of knowing this beforehand).

For more details on number 2, see [the testing library docs](https://testing-library.com/docs/queries/about#priority) on query priorities which we've largely tried to follow in the past.

Reply:
For #2, I was able to use queryByLabelText after adding aria-labelledBy to the information icon, which is actually the correct way to do accessibility for tooltip :)

Reply:
Awesome. And TIL how to correctly do accessibility for tooltip.